docs/validation-rules.rst

```diff
@@ -457,6 +458,22 @@ minlength, maxlength
 
 Minimum and maximum length allowed for iterables.
```

Comment:
on that occassion you could correct one of my mistakes: "iterables" is false in that sentence. truth is that any object that implements `__len__` works with this rule. the abstract class is `collections.abc.Sized`, but i don't think the term "sized" is established and clear.

Reply:
True that in my project to we had a bit of trouble with collections as we were trying to validate Cerberus Schema and on first thought I just went to validate if the schema is a dict or not. But, later after digging into Mapping and Cerberus's source code that I realized that any Mapping type can be validated.  

I have changed it now let me know if this works.